### PR TITLE
Prettier Update

### PR DESCRIPTION
**Pull request by nataliepina**
In this PR, I have updated Prettier, and set a script to run the formatting command. We needed a .prettierrc file for this to work and to have some rules to go off of, I've used the recommended Prettier rules along with some custom settings. I didn't want to cause too many changes with the custom settings so I've turned off the "commaDangle" setting for now. This can be addressed on a separate PR. 

The enzyme adapter needed an update to fix some failing tests. 

The second commit is dedicated to the result of running `npm run format` and is mostly to update spacing and small fixes. I've set things so that it won't touch too many files so that it won't destroy important parts of our git history. 

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

**Review by mstruebing — APPROVED** [review]
LGTM 👍

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
[the remark this answers is not in the document]
Comment (nataliepina) [reply .prettierrc:1]: Oops, renamed the file and it kept the old one it seems. Updated.

**Comment by seialkali** [comment]
Thanks for updating this and adding it to ESLint. I've noticed that these changes will alert us to Prettier formatting issues via ESLint, but it won't stop us from committing or pushing changes with Prettier issues. What are your thoughts on setting up something similar to what we have in kommander-ui where it will do a Prettier format on commit?

**Comment by nataliepina** [comment]
> Thanks for updating this and adding it to ESLint. I've noticed that these changes will alert us to Prettier formatting issues via ESLint, but it won't stop us from committing or pushing changes with Prettier issues. What are your thoughts on setting up something similar to what we have in kommander-ui where it will do a Prettier format on commit?

Yes! I was definitely considering a format on lint. I'm thinking about using husky for this since I've used it in the past but am open to whatever. If you have anything in mind you can go for it too, if you'd like!

**Review by TattdCodeMonkey — APPROVED** [review]

**Pull request merged**